builder/install/install-dependencies.sh

```diff
@@ -7,7 +7,7 @@ export LICENSE_DIR="/THIRD_PARTY_NOTICES"
 mkdir -p "${LICENSE_DIR}"
 
 export NPROCS
-NPROCS="$(nproc)"
+NPROCS="${NPROCS:-$(nproc)}"
```

Comment:
Let maybe not rely on nproc at all, and just have a constant default, 4 or something?

Reply:
The builder build is extermely slow as it is, if I can build it with nproc locally I want to be able to do that. This way we can just cap it on the systems that are giving us trouble, do you see any problems with this approach?

Reply:
Also, GHA runners have just 2 vCPUs if I recall correctly, so capping it to 4 also doesn't sound quite right.